my-app/src/file.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import {SketchField, Tools} from 'react-sketch';
+
+import Chat from './chat'
+import 'react-sliding-pane/dist/react-sliding-pane.css';
+
+class File extends React.Component{
+	
+
+	render(){
+		return(
+			<div>
+				<div style = {{height: 70, backgroundColor: 'white'}}>
+					<h3 style= {{marginLeft: 15}}>Filename</h3>
```

Comment:
This component is missing a button to click to get back to the main page with the list of all the rooms

Also missing an audio chat button